react-app/src/components/ResultPage.js

```diff
@@ -14,7 +14,7 @@ function ResultPage(props) {
     <div>
       <Container fluid>
         <Row>
-          <Col xxl={3} xl={3} lg={3} md={3} sm={3} xs={3} className="sidebar-col">
+          <Col xxl={1} xl={1} lg={1} md={1} sm={1} xs={3} className="sidebar-col">
```

Comment:
DP - Could you look at the grid system documentation closer and see if there is a way to reduce the number of breakpoints we use here?

Reply:
It looks like I can delete the xxl, xl, lg, and md breakpoints and have the same effect